spynnaker/pyNN/models/neural_projections/connectors/multapse_connector.py

```diff
@@ -69,9 +79,8 @@ def _update_synapses_per_post_vertex(self, pre_slices, post_slices):
             prob_connect = [
                 float(pre.n_atoms * post.n_atoms) / float(n_connections)
                 for pre in pre_slices for post in post_slices]
-            self._synapses_per_edge = self._rng.next(
-                1, distribution="multinomial", parameters=[
-                    self._num_synapses, prob_connect])
+            self._synapses_per_edge = self.get_rng_next(
```

Comment:
Quick test - this doesn't seem to exist...

Reply:
It had to move out to sPyNNaker7/8 as the "multinomial" keyword was removed between PyNN 0.7 and 0,8.  I believe they plan to add this back in again but for now I've split it up until they actually fix it.  (It's not in 0.9.2 yet as far as the documentation says...)

Reply:
Ahh OK, I missed this.  Best thing to do then is to add this as an @abstractmethod to show that you know that it isn't defined here.